tests/test_util.py

```diff
@@ -25,52 +27,53 @@ def test_escape(self):
         assert escape_literal([1, 'a', date(2001, 1, 1)]) ==\
             "(1, 'a', toDate('2001-01-01'))"
 
-    def test_conditions_expr(self):
+    @pytest.mark.parametrize('dataset', DATASETS)
```

Comment:
This one I don;t think will ever work on any dataset other than events.
See tags

Reply:
Yeah, some of these will need to be rewritten for new datasets